packages/interface/src/components/interface-skeleton/style.scss

```diff
@@ -128,6 +128,7 @@ html.interface-interface-skeleton__html-container {
 
 .interface-interface-skeleton__header {
 	flex-shrink: 0;
+	min-height: 37px;
```

Comment:
Why 37px? Should we have this valued based on the css variables we have?

Reply:
I have no idea what this change is about, I guess we can try to remove it and see what happens :)